frontend/src/widgets/QualityAssuranceDashboardOverview.js

```diff
@@ -0,0 +1,207 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Grid, Tooltip as TrussWorksToolTip } from '@trussworks/react-uswds';
+import { Link } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faPersonChalkboard,
+  faBus,
+  faUser,
+  faQuestionCircle,
+} from '@fortawesome/free-solid-svg-icons';
+import './QualityAssuranceDashboardOverview.scss';
+
+import Loader from '../components/Loader';
+import Tooltip from '../components/Tooltip';
+import colors from '../colors';
+
+export function Field({
```

Comment:
Can we please reuse existing components here, and just add a prop to handle the differences? that way we wouldn't need a whole new CSS file/reinvent the wheel

Reply:
Yea I added a new component and used it in both places.